cEP-0021.md

```diff
@@ -0,0 +1,379 @@
+# Debug and Profile Bears
+
+| Metadata |                                                   |
+| -------- | ------------------------------------------------- |
+| cEP      | 0019                                              |
+| Version  | 1.0                                               |
+| Title    | Debug and Profile Bears                           |
+| Authors  | Vaibhav kumar Rai <mailto:raivaibhav08@gmail.com> |
+| Status   | Proposed                                          |
+| Type     | Process                                           |
+
+## Abstract
+
+This cEP describes the detailed implementation of profiling interface as a part
```

Comment:
why only profiler? what about the debugger, profiling is the shorter part of the two right?

Reply:
Check this out 
>The applicant has made a cEP of how the profiling will be implemented has been merged.

https://projects.coala.io/#/projects?project=debug_and_profile_bears&lang=en